package.json

```diff
@@ -2,11 +2,12 @@
   "name": "react-homework-template",
   "version": "0.1.0",
   "private": true,
-  "homepage": "https://goitacademy.github.io/react-homework-template/",
+  "homepage": "https://romanchenko31.github.io/reactHomework-1/",
   "dependencies": {
     "@testing-library/jest-dom": "^5.16.3",
     "@testing-library/react": "^12.1.4",
     "@testing-library/user-event": "^13.5.0",
+    "clsx": "^1.2.1",
```

Comment:
why do you need that package, I do not see that you use it

Reply:
hz. i wanted use it. but i changed my mind

Reply:
i remove it